treeherder/etl/job_loader.py

```diff
@@ -50,21 +50,25 @@ def process_job_list(self, all_jobs_list):
 
         for project, job_list in validated_jobs.items():
             newrelic.agent.add_custom_parameter("project", project)
-            repository = Repository.objects.get(name=project)
-
-            storeable_job_list = []
-            for pulse_job in job_list:
-                if pulse_job["state"] != "unscheduled":
-                    try:
-                        self.clean_revision(repository, pulse_job)
-                        storeable_job_list.append(
-                            self.transform(pulse_job)
-                        )
-                    except AttributeError:
-                        logger.warn("Skipping job due to bad attribute",
-                                    exc_info=1)
-
-            store_job_data(repository, storeable_job_list)
+            try:
+                repository = Repository.objects.get(name=project)
+
+                storeable_job_list = []
+                for pulse_job in job_list:
+                    if pulse_job["state"] != "unscheduled":
+                        try:
+                            self.clean_revision(repository, pulse_job)
+                            storeable_job_list.append(
+                                self.transform(pulse_job)
+                            )
+                        except AttributeError:
+                            logger.warn("Skipping job due to bad attribute",
+                                        exc_info=1)
+
+                store_job_data(repository, storeable_job_list)
+
+            except Repository.DoesNotExist:
+                logger.warn("Job with unsupported project: {}".format(project))
```

Comment:
s/warn/info/ for `Repository.DoesNotExist`? (I agree the `AttributeError` case should be warn)